sdk/core/amqp-common/package.json

```diff
@@ -106,11 +107,11 @@
     "lint": "eslint -c ../../.eslintrc.json src test samples --ext .ts -f node_modules/eslint-detailed-reporter/lib/detailed.js -o amqp-common-lintReport.html || exit 0",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
-    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run build:test && npm run unit-test && npm run integration-test",
-    "unit-test:browser": "karma start --single-run",
-    "unit-test:node": "cross-env TS_NODE_FILES=true TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\":\\\"commonjs\\\"}\" nyc --reporter=lcov --reporter=text-lcov mocha -r ts-node/register -t 50000 --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- ./test/**/*.spec.ts",
+    "test:browser": "npm run build:test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test:node && npm run integration-test:node",
+    "test": "npm run test:node && npm run test:browser",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "echo skipped",
```

Comment:
AFAIK no tests in amqp-common are live/integration tests.
The format we have been following is
- `unit-test:*` is for tests that dont hit live service, this gets run with each PR
- `integration-test:*` for tests that hit live service, this gets run nightly
- `test:` for dev inner loop

So, for amqp-common, I would imagine that 
- `integration-test:*` tests would be `echo skipped`.
- `unit-test:*` test would run the tests that we have
- `test:*` would build code and run the unit tests